### PR TITLE
Disable various flaky tests.

### PR DESCRIPTION
**Pull request by petermattis**
See #3092, #4378, #4445, #4500, #4501, #4512, #4517.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4519)

### CODE REVIEW

File: gossip/storage_test.go [hunk 1]
```diff
@@ -89,6 +89,7 @@ func (s unresolvedAddrSlice) Swap(i, j int) {
 // using the bootstrap hosts in a gossip.Storage object.
 func TestGossipStorage(t *testing.T) {
 	defer leaktest.AfterTest(t)
+	t.Skipf("TODO(pmattis): #4500")
```
Comment (tamird) [inline gossip/storage_test.go]: the TestGossipStorage part of that issue is fixed

**Comment by dt** [comment]
Reviewed 7 of 7 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4519)*

File: acceptance/gossip_peerings_test.go [hunk 2]
```diff
@@ -141,6 +141,8 @@ func TestGossipPeerings(t *testing.T) {
 // re-bootstrapped after a time when all nodes were down
 // simultaneously.
 func TestGossipRestart(t *testing.T) {
+	t.Skipf("TODO(pmattis): #4445")
```
Comment (tamird) [inline acceptance/gossip_peerings_test.go]: i dont think this is flaky in CI since we don't run 5 nodes there

**Comment by tamird** [comment]
LGTM

File: gossip/storage_test.go (hunk 1 above)
Comment (petermattis) [inline gossip/storage_test.go]: Ok, I'll revert this.

File: acceptance/gossip_peerings_test.go (hunk 2 above)
Comment (petermattis) [inline acceptance/gossip_peerings_test.go]: It failed for me on CI 15 hours ago: https://circleci.com/gh/petermattis/cockroach/321?utm_campaign=build-failed&utm_medium=email&utm_source=notification

**Comment by BramGruneir** [comment]
LGTM

File: acceptance/gossip_peerings_test.go (hunk 2 above)
Comment (tamird) [inline acceptance/gossip_peerings_test.go]: Spencer fixed it in #4498.

Comment (tamird) [inline acceptance/gossip_peerings_test.go]: oh, wait, misread; carry on

Comment (petermattis) [inline acceptance/gossip_peerings_test.go]: Yeah, it is still flaky (I think).

Comment (spencerkimball) [inline acceptance/gossip_peerings_test.go]: Yeah there's no good reason this should fail with 5 but not 3 nodes. Probably 5 just increases the chance that whatever is going wrong goes wrong.

Comment (BramGruneir) [inline acceptance/gossip_peerings_test.go]: I think it's the same root cause as the flakiness for the status server test as well (#4512).

**Pull request merged**